packages/reports/addon/components/navi-column-config.js

```diff
@@ -83,6 +83,19 @@ class NaviColumnConfig extends Component {
         fragment: dimension
       };
     });
+
+    if (timeGrain !== 'all') {
+      dimensionColumns.unshift({
+        type: 'dimension',
```

Comment:
Since carbon/elide (somewhat) recently added the `TimeDimension` column type, we should adopt it in the request v2 spec and set the type here. Would that cleanup some logic?